content/dotnet-template-azure-iot-edge-module/CSharp/SampleModule.csproj

```diff
@@ -15,12 +11,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.*" />

```

Comment:
I'd recommend against using wildcards. You can't guarantee that an update won't have broken the sample. Better to have it released in a known working state against a specific version.

Reply:
I agree, although this has been the strategy with the template, so there is no need to update when there is a new Device.Client version. BTW, SemVer 1.* will ensure there is no breaking changes.

Reply:
`will ensure`? More like should ensure. We've had unintentional breaking changes.

Reply:
Ok, updated